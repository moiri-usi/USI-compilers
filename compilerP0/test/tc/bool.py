print 1, True
print 2, False
print 3, True + 2
print 4, False - True
print 5, False - False + True + False - True
print 6, input()
