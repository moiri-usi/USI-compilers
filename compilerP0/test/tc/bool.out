1 True
2 False
3 3
4 1
5 0
6 True
